mergebot_backend/mergebot_poller.py

```diff
@@ -125,6 +125,10 @@ def poll(self):
                     self.publisher.publish_poller_status(status='SHUTDOWN')
                     return
             self.publisher.publish_poller_heartbeat()
```

Comment:
I've taken a note to look into this. It may work to return a tuple of (map, err) and set the authorized users list only if `not err`